deploy/svg_generator/svg_generator.py

```
@@ -131,6 +135,10 @@ async def get_weather(mc, key_b, default_response={}):
     return await get_cache_data(mc, key_b, default_response={})
 
 
+async def get_energy(mc, key_b, default_response={}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)